package.json

```diff
@@ -28,7 +28,7 @@
     "test-cli": "yarn --cwd 'acceptance-tests' test-ci",
     "test-cli-debug": "yarn --cwd 'acceptance-tests' test-debug",
     "test-cli-qa": "yarn --cwd 'acceptance-tests' test-qa",
-    "test-docker": "yarn build-docker && docker container run -it --rm --name=hs-cli-container hs-cli-image yarn test",
+    "test-docker": "yarn build-docker && docker container run -it --rm --name=hs-cli-container hs-cli-image yarn test-cli",
```

Comment:
This was set to run the unit tests and I changed it to use the ATs instead.  I'm not sure why we would want to run the unit tests in a docker container, but I can add another command if we would like to.